### PR TITLE
added docs

### PR DESCRIPTION
**Pull request by alekseytsekov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,159 @@
...
+The backend will be used to join needed data(amount of tokens, merkle tree leaf index and an array of intermediary hashes) from provided user ethereum address, AE address and signature(signed AE address). With this data, backend would create contract call tx, signed it with internal wallet and broadcasted it. If smart contract accept(verified) current request would transfer tokens to provided AE address, emit event and would return number of transfer. After that backend would notify backendless service for current transfer and user would not be able to transfer it again.
+
+### Installation
+What you should have installed 
+ - Go: https://golang.org/dl/
+ - local postgresql(https://www.postgresql.org/download/) or connect to remote one
+
+clone this repo
+in project root directory `go get`
+set environment variables
```
Comment (Perseverance) [inline README.md]: Can you give an example of the environment variables that need to be set. Also it is a good idea to tell where they shall be set

**Review by Perseverance — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alekseytsekov) [reply README.md]: there is '.env_example' file - https://github.com/LimeChain/aepp-token-migration-backend/blob/fix/move-credentials-to-env/.env_example

Comment (Perseverance) [reply README.md]: Yep, lets just say in the readme - `set the environment variables in an .env file. You can see example in the .env_example file`

File: README.md [hunk 2]
```diff
@@ -1 +1,222 @@
...
+
+## Backend
+The backend will be used to join needed data(amount of tokens, merkle tree leaf index and an array of intermediary hashes) from provided user ethereum address, AE address and signature(signed AE address). With this data, backend would create contract call tx, signed it with internal wallet and broadcasted it. If smart contract accept(verified) current request would transfer tokens to provided AE address, emit event and would return number of transfer. After that backend would notify backendless service for current transfer and user would not be able to transfer it again.
```
Comment (HristiyanG) [inline README.md]: With this data, backend would create contract call tx, // signed => sign // it with internal wallet and // broadcasted => broadcast // it. If smart contract accept => accepts (verified => verifies)

File: README.md [hunk 3]
```diff
@@ -1 +1,222 @@
...
+        "C9761373ADDBEEEB73C932B213985FF233C3207D873394D5AF8E8071EA823EA2",
+        "AB0FA2604F5F394B0ADFC2AAE55B8296F390A58910E7CC6E1DC031023941A259",
+        "9FEF092769B79A4ED4540D43386A4EA6A6B8F03A89DB2E3B7F955817EBA05ED4",
+        "16D23EC2BE773B1ABD000772D8FAE6D7502252381E3B21853697FDD0BA0D78B9"
+    ]
+}
+```
+
+
+`/validate` - POST validate is provided data can generate same merkle root hash. Should pass ethereum address, amount of tokens, leaf index and an array of intermediary hashes. Returns is exists
```
Comment (HristiyanG) [inline README.md]: Returns is exists => 
Returns object with boolean property ‘exists’ which validates if the request is valid or not.

**Review by HristiyanG — CHANGES_REQUESTED** [review]

**Review by HristiyanG — APPROVED** [review]

File: .env_example [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
+AE_COMPILER_URL_PROD="xxxxxxx"
+
+AE_NETWORK_ID="ae_devnet"
+AE_NETWORK_ID_PROD="ae_devnet"
+# AE_NETWORK_ID_PROD="ae_uat"
+
+AE_BACKEND="fate"
+AE_ABI_VERSION=3
+
+BACKENDLESS_LOGIN="xxxxxx"
```
Comment (maptuhec) [inline .env_example]: again, do we need this in the exampple?

File: README.md [hunk 5]
```diff
@@ -179,4 +196,27 @@ response:
 }
 ```
 
+## Import token owners (db table backup)
+
+
+1. install brew (https://brew.sh/) `if you have ‘psql’ as env variable skip this step`
```
Comment (maptuhec) [inline README.md:207]: is this valid for windows users?

File: .env_example (hunk 4 above)
Comment (alekseytsekov) [reply .env_example]: I removed it in another PR, now removed in this ;)

**Pull request merged**